nucypher/utilities/sandbox/blockchain.py

```diff
@@ -48,10 +58,12 @@ class TesterBlockchain(Blockchain):
     Blockchain subclass with additional test utility methods and options.
     """
 
+    _PROVIDER_URI = 'tester://pyevm'
     _instance = NO_BLOCKCHAIN_AVAILABLE
     _test_account_cache = list()
+    _default_test_accounts = 10
 
-    def __init__(self, test_accounts=None, poa=True, airdrop=True, *args, **kwargs):
+    def __init__(self, test_accounts=_default_test_accounts, poa=True, airdrop=True, *args, **kwargs):
```

Comment:
This is an odd pattern - what made you decide to do this?

Reply:
To add the `10`?  Just because we always expect there to be => 10 accounts in tests.

Reply:
I meant the pattern of resolution here - you have `_default_test_accounts`, a class variable, but which is only ever used at class evaluation time to set the default of `__init__`... why not just either make it `10` in `__init__` or have `__init__` grab the class variable?

Reply:
Hmm.. I *think* I want this at the instance level ultimately.  :heavy_check_mark: